lib/ansible/modules/windows/win_firewall_rule.ps1

```diff
@@ -172,18 +172,22 @@ $action = Get-AnsibleParam -obj $params -name "action" -type "str" -failifempty
 $program = Get-AnsibleParam -obj $params -name "program" -type "str"
 $service = Get-AnsibleParam -obj $params -name "service" -type "str"
 $enabled = Get-AnsibleParam -obj $params -name "enabled" -type "bool" -default $true -aliases "enable"
-$profiles = Get-AnsibleParam -obj $params -name "profiles" -type "str" -default "domain,private,public" -aliases "profile"
+$profiles = Get-AnsibleParam -obj $params -name "profiles" -type "list" -default @("domain", "private", "public") -aliases "profile"
 $localip = Get-AnsibleParam -obj $params -name "localip" -type "str" -default "any"
 $remoteip = Get-AnsibleParam -obj $params -name "remoteip" -type "str" -default "any"
 $localport = Get-AnsibleParam -obj $params -name "localport" -type "str"
 $remoteport = Get-AnsibleParam -obj $params -name "remoteport" -type "str"
 $protocol = Get-AnsibleParam -obj $params -name "protocol" -type "str" -default "any"
-$interfacetypes = Get-AnsibleParam -obj $params -name "interfacetypes" -type "str" -default "any"
+$interfacetypes = Get-AnsibleParam -obj $params -name "interfacetypes" -type "list" -default @("any")
 $edge = Get-AnsibleParam -obj $params -name "edge" -type "str" -default "no" -validateset "no","yes","deferapp","deferuser"
 $security = Get-AnsibleParam -obj $params -name "security" -type "str" -default "notrequired" -validateset "notrequired","authnoencap","authenticate","authdynenc","authenc"
 
 $state = Get-AnsibleParam -obj $params -name "state" -type "str" -default "present" -validateset "present","absent"
+
 $force = Get-AnsibleParam -obj $params -name "force" -type "bool" -default $false
+if ($force) {
+    Add-Warning -obj $result -message "'force' isn't required anymore"
```

Comment:
I probably should have given you a different comment before but `Add-DeprecatedWarning` would be better here. This way we can eventually remove this parameter in version 2.9. In the end it should look something like this

`Add-DeprecatedWarning -obj $result -mesage "blah" -version 2.9`

It would also be good to update the module docs to say this is deprecated and will be removed in 2.9.